core/internal/gethwrappers/go_generate_test.go

```diff
@@ -39,7 +36,7 @@ func TestCheckContractHashesFromLastGoGenerate(t *testing.T) {
 				"changes", wd)))
 	*/
 	for _, contractVersionInfo := range versions.ContractVersions {
-		if isOCRContract(contractVersionInfo.CompilerArtifactPath) {
+		if isOCRContract(contractVersionInfo.AbiPath) {
```

Comment:
How do we ensure that the OCR abi is constructed from the latest version from libocr?

Reply:
Do we have to? I imagine the ABI being very stable. 
Another approach might be to include the abijson files in the libocr repo